openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -719,6 +719,15 @@
       // genereal feedback question
       .assessment__rubric__question--feedback {
 
+        .wrapper--input {
+          margin-top: $baseline-v;
+        }
+
+        .question__title__copy {
+          @include margin-left(0);
```

Comment:
Instead of adding the override, is it possible to remove the thing that caused the indent? Right now I see various places where question__title__copy is associated with the left margin, and then overwritten with a margin of 0.

When is the left margin desirable? Are all the places where question__title__copy appears in SASS actually necessary?